docs/resources.rst

```diff
@@ -71,22 +60,31 @@ Editor Integrations
 
 * Vim
 
-    * `Vim Solidity <https://github.com/tomlion/vim-solidity/>`_
+    * `Vim Solidity by Thesis <https://github.com/thesis/vim-solidity/>`_
+        Syntax highlighting for Solidity in Vim.
+
+    * `Vim Solidity by tomlion <https://github.com/tomlion/vim-solidity/>`_
         Plugin for the Vim editor providing syntax highlighting.
 
     * `Vim Syntastic <https://github.com/vim-syntastic/syntastic>`_
         Plugin for the Vim editor providing compile checking.
 
 * Visual Studio Code
 
-    * `Visual Studio Code extension <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
+    * `Ethereum Remix <https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix>`_
+        Ethereum Remix plugins in VSCode
+
+    * `Solidity by Juan Blanco <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
```

Comment:
I think these names look confusing without having something like "extension" or "for VS Code" in them.
```suggestion
    * `Ethereum Remix extension <https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix>`_
        Ethereum Remix plugins in VSCode

    * `Solidity extension by Juan Blanco <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
```

Reply:
Maybe the name itself can be in a different format. For example: 

```md
`Solidity` extension by Juan Blanco
```

or 

```md
`Solidity` Visual Studio Code extension, by Juan Blanco
```

Reply:
By the way, is it Hardhat for VSC or Solidity for VSC? :)

Reply:
All of these are in the `Visual Studio Code` so adding VSC/VS Code, etc. may be somewhat redundant (to play the devil's advocate).

Reply:
I think I prefer something like

```
Solidity Visual Studio Code extension, by Juan Blanco
```

just because we also call the other extensions by their name (e.g. "Vim Solidity" or "Emacs Solidity" and not only "Solidity"). I think it adds value to keep the Visual Studio Code, even if it's repetitive (since it's all under the VS Code bullet point).

Reply:
Also, **extension pack** is the correct nomenclature on VS Code world.
```
 * `Ethereum Remix, by the Remix Project <https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix>`_
	        Ethereum Remix extension pack for VS Code
```

Reply:
Can I suggest using the contracted `VS Code extension` instead of `Visual Studio Code extension`, so:

```
* Ethereum Remix VS Code extension ...
* Solidity VS Code extension, by Juan Blanco ...
* Solidity VS Code extension, by Nomic Foundation ...
* Solidity Visual Auditor VS Code extension ...
* Truffle for VS Code ...

Leaving Truffle as is because it is close enough?

```

Reply:
> extension pack is the correct nomenclature on VS Code world

I think that's one possible category of extensions.

> By the way, is it Hardhat for VSC or Solidity for VSC? :)

We renamed to Solidity, as we are added general support for Solidity, including, for example, in Foundry projects.

Reply:
> We renamed to Solidity, as we are added general support for Solidity, including, for example, in Foundry projects.

Looking at it from user's perspective, I think that Hardhat would still be clearer to me on this list, but well, if that's your preferred official naming then I guess we should follow that.